src/components/Map/USACountyMap.tsx

```diff
@@ -0,0 +1,204 @@
+ * box (with risk level color) + text.
+ *
+ * This is special cased from the normal map display. The mariana islands are
+ * small enough that simply showing the islands is not a UX that works.
+ */
+const MarianaIslands = ({
+  fill,
+  onMouseEnter,
+  onMouseLeave,
+  onClick,
+  tabIndex,
+}: {
+  fill: string;
+  onMouseEnter: () => void;
+  onMouseLeave: () => void;
+  onClick: () => void;
+  tabIndex: number;
+}) => {
+  return (
+    <g
+      transform="translate(40, 395) scale(0.8)"
+      onMouseEnter={onMouseEnter}
+      onMouseLeave={onMouseLeave}
+      onClick={onClick}
+      tabIndex={tabIndex}
+    >
+      <rect width="20" height="20" fill={fill} />
+      <text transform="translate(25, 15)">CNMI</text>
+    </g>
+  );
+};
+
+interface USACountyMapProps {
+  stateClickHandler: (stateName: string) => void;
+  setTooltipContent: (content: string) => void;
+  showCounties: boolean;
+}
+
+const USACountyMap = React.memo(
+  ({
+    stateClickHandler,
+    setTooltipContent,
+    showCounties,
+  }: USACountyMapProps) => {
+    const locationSummaries = useSummaries();
+
+    const getFillColor = (geo: any) => {
+      const summary = (locationSummaries && locationSummaries[geo.id]) || null;
+      return colorFromLocationSummary(summary);
+    };
+
+    const projection = geoAlbersUsaTerritories()
+      .scale(1070)
+      .translate([400, 300]);
+
+    const onMouseLeave = () => setTooltipContent('');
+
+    return (
+      <USMapWrapper>
+        {/** Map with shaded background colors for states. */}
+        <USStateMapWrapper $showCounties={showCounties}>
+          <ComposableMap data-tip="" projection={projection} height={500}>
+            <g transform="translate(0, -50)">
+              {showCounties && (
+                <Geographies geography={geoCounties}>
+                  {({ geographies }) =>
+                    geographies.map(geo => {
+                      return (
+                        <Geography
+                          key={geo.rsmKey}
+                          geography={geo}
+                          fill={getFillColor(geo)}
+                          strokeWidth={0}
+                          role="img"
+                        />
+                      );
+                    })
+                  }
+                </Geographies>
+              )}
+              <Geographies geography={geoStates}>
+                {({ geographies }) =>
+                  geographies
+                    .filter(geo => stateFipsCodes.includes(geo.id))
+                    .map(geo => {
+                      const fipsCode = geo.id;
+                      const state = regions.findByFipsCode(fipsCode);
+
+                      if (
+                        state === null ||
+                        !Boolean(state) ||
+                        !(state instanceof StateType)
+                      ) {
+                        return null;
```

Comment:
Rather than trying to have all of this be conditional on something that shouldn't happen, I think you can use `findByFipsCodeStrict` and we'll just crash if the state is missing.

The old code would have already crashed in this case, since it was using `state.stateCode` before checking if `state` was defined.

Reply:
agree, I'll just coerce to a state and eliminate the if check